.ci/Jenkinsfile-extension-vsce

```diff
@@ -92,6 +92,15 @@ pipeline {
                             sed -i "1s/\\?/${release_date}/" CHANGELOG.md
                             git commit -m "set release date: ${release_date}" CHANGELOG.md
 
+                            # add a requirements.txt file to tie the extenstion to a server version
+                            if [[ "${LS_version}" == "latest" ]]; then
+                                ${WORKSPACE}/env/bin/pip install inmantals
+                            else
+                                ${WORKSPACE}/env/bin/pip install inmantals=="${LS_version}"
+                            fi
+                            ls_version=$(${WORKSPACE}/env/bin/pip freeze | grep inmantals | cut -d'=' -f3
+                            echo "inmantals~=${ls_version}" > requirements.txt
```

Comment:
Two remarks (I should probably have given a short introduction as to how this pipeline works because it's quite niche):
- you'll need to commit this
- I don't think we want this on the `issue/version-bump-extension` branch (which gets merged both into master and next eventually) but only on the `next` branch, so you'll need to move this down a bit.